Makefile

```diff
@@ -15,6 +15,10 @@ up-all: ## Start the docker containers (including forwarder)
 	docker-compose --profile all up
 	docker-compose logs -f
 
+.PHONY: env 
+env: ## Copy alls env sample files to .env files
+	for file in $$(find . -name ".env.*.sample"); do cp $${file} $${file%.*}; done 
```

Comment:
I don't understand what this is useful for. 🤔

Reply:
Okay, the README made it a little clearer. Maybe we should change the help message. How about "Create required .env files from sample files"?

Reply:
changed